src/functions.js

```diff
@@ -18,22 +18,21 @@ const add = () => {
   task.id = tasksArray.length;
   tasksArray.push(task);
   updateStorage(tasksArray);
-  console.log(tasksArray);
 };
 
-// Remove completed tasks function (i did not realize this was for the next project)
-
-// const removeCompTask = () => {
-//   tasksArray = tasksArray.filter((task) => task.completed !== true);
-//   updateStorage(tasksArray);
-// };
-
 // remove a task from the array
 const removeTask = (id) => {
   tasksArray = tasksArray.filter((task) => task.id !== id);
   updateStorage(tasksArray);
 };
 
+// function to changed the status of the task
+const toggleCompleted = (id) => {
```

Comment:
- [ ] Dear @adeola003, Good job making most of the functionalities. But there is some issue with your current application we need to address. As you can see, in your application when we add and remove data the do-to list id/index is not getting updated. as a result the remove and clear all functionality does not work properly. Please make sure every task has a unique id as it enters and the index gets updated as we delete the task. This will make sure you remove functionality only removes the appropriate task.

https://user-images.githubusercontent.com/69077061/229220682-53d5c816-4083-4bb3-bff6-7f5ca732fa95.mp4